status-im/status-react
# [#21557] feat: show from account page in swap flow

fixes #21557

### Summary

The ["From" page](https://www.figma.com/design/AD2JSKg0I8dZcylyiGa62O/Swap-for-Mobile?node-id=1691-48512&node-type=instance&m=dev) should appear, allowing the user to choose the account for the swap.

#### Areas that maybe impacted

- Swap flow when user has multiple account



### Steps to test

1. Ensure user has at least 2 accounts set up.
2. Go to the wallet main page.
3. Tap on "Swap" for any asset.


### Result


https://github.com/user-attachments/assets/adc21d9d-3140-41ca-bfa8-51152b19e961



status: ready
